services/open-vsx/open-vsx-version.tester.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+const t = (module.exports = require('../tester').createServiceTester())
+const { withRegex } = require('../test-validators')
+
+const isVersion = withRegex(/^v(\d+\.\d+\.\d+)(\.\d+)?$/)
```

Comment:
Since Open VSX is alternative for VS Marketplace, I'm basing my work using the `visual studio marketplace` service. Here, I actually copied the regex used by `visual studio marketplace` service to test its version:
https://github.com/badges/shields/blob/71299a81788c95363bbafc4a2c75af960b12d5b9/services/visual-studio-marketplace/visual-studio-marketplace-version.tester.js#L5-L7

The `isVPlusDottedVersionNClauses` wouldn't be suffice for this, since VS extension version needs to be at least in three part `MAJOR.MINOR.PATCH` with an optional `BUILD` (4th part).

Reply:
gotcha. it may be worthwhile in a follow up PR if you (or anyone else) is interested to pull this out into a separate validator and update these testers to utilize it instead of duplicating it, but fine as-is for this PR :+1: